NOTICE.txt

```diff
@@ -1,6 +1,6 @@
 CCPi Core Imaging Library (CIL).
-Copyright 2017-2021 Rutherford Appleton Laboratory STFC
-Copyright 2017-2021 University of Manchester
+Copyright 2017-2022 Rutherford Appleton Laboratory STFC
```

Comment:
```suggestion
Copyright 2017-2022 United Kingdom Research and Innovation
```

Note in the CIL Astra files, which we updated after discussing with the lawyers, we have:
https://github.com/TomographicImaging/CIL/blob/00b55e453fb6e8f81be296f09bd059133f46c885/Wrappers/Python/cil/plugins/astra/operators/AstraProjector2D.py#L2-L3

I also wonder if next to our names it should be (STFC, UKRI)